packages/DatePicker/src/DatePicker.js

```diff
@@ -0,0 +1,166 @@
+import React from "react";
+import PropTypes from "prop-types";
+import moment from "moment";
+import momentPropTypes from "react-moment-proptypes";
+
+import CalendarIcon from "@paprika/icon/lib/Calendar";
+import Input from "@paprika/input";
+import Popover from "@paprika/popover";
+import useI18n from "@paprika/l10n/lib/useI18n";
+
+import Calendar from "./components/Calendar/Calendar";
+import DateInput from "./components/DateInput/DateInput";
+
+import { CalendarStyles } from "./DatePicker.styles";
+import { getExtendedInputProps } from "./helpers";
+
+const propTypes = {
+  children: PropTypes.node,
+
+  /** Selected date in moment object. */
+  date: momentPropTypes.momentObj,
```

Comment:
does this need to be a moment? cannot be a string? could you received a string and then parse with the format? so the consumer doesn't need to install moment in their side?

Reply:
Sounds good! Does it mean we'll also output `string` instead of `momentObj` in `onChange` callback? or maybe output both?

Reply:
yes! lets response with a string. I think they could parse to moment again, the consumer is passing the format from the beginning, so they might be able to do it by themselves without our help.

Reply:
`react-dates` expects to be passed a moment object though. So we'll need to parse the date into moment, do we know how to do this correctly? (i.e. accounting for locale/format)

Reply:
@alexzherdev I think @nahumzs 's idea is based on the format passed in or by the default format.

@nahumzs Sometimes the date format to display is different than the format used in the data flow. For example, the server requires date string in `YYYY-MM-DD` format, but UI requirement is displaying `DD/MM/YYYY` in the input. As the developer, I'll pass in `DD/MM/YYYY` in `format` prop like this: `<DatePicker format="DD/MM/YYYY">`. If the output in `onChange` function is string, I have to convert it to moment obj, and convert it back to server format string, that's two steps. So I still prefer return 2 types of date data in the `onChange` function.

Reply:
One of the problem when using `momentPropTypes.momentObj` is that will yield is an invalid prop value if you pass `null` or `""` which is really annoying because something you can't control that. That's why I'm asking if it's possible to find a different way to avoid using momentPropTypes.momentObj at all. 

How would you mark that you want a `<DatePicker />` without a date?

Reply:
I talked with @allison-c let's keep moment propTypes even if it's annoying that you can pass `data=""` as an empty string.